Botkit/Microsoft.BotKit/Conversation/BotkitConversation.cs

```diff
@@ -1,46 +1,47 @@
-// Copyright(c) Microsoft Corporation.All rights reserved.
+﻿// Copyright(c) Microsoft Corporation.All rights reserved.
 // Licensed under the MIT License.
 
-using Microsoft.BotKit.Core;
-using Microsoft.Bot.Builder.Dialogs;
-using System;
-using System.Threading.Tasks;
-
 namespace Microsoft.BotKit.Conversation
 {
+    using System;
+    using System.Threading.Tasks;
+    using Microsoft.Bot.Builder.Dialogs;
+    using Microsoft.BotKit.Core;
```

Comment:
We are ignoring the rule regarding the _using_ inside a _namespace_. Can you revert this change?